### ISSUE TITLE
postcss 5.x support

### ISSUE THREAD

**Issue by pirxpilot** [code42day/postcss-cli#24]
Branch that supports postcss 5.x is [here](https://github.com/code42day/postcss-cli/tree/support-postcss-5)

Among other things it removes support for `--safe` option. And it requires node >= 0.12
Check it out, send feedback, patches etc. I plan to release it as postcss-cli 2.0.0 if everything works OK.

@FND - if you find a moment could you check if `--watch` works OK. Thanks!

**Comment by FND** [comment]
Yup, `--watch` still seems to work alright. However:
- postcss-import doesn't [officially support](https://github.com/postcss/postcss-import/blob/5db759d7b9f63075f3150263f2d6d89a1d74149f/package.json#L29) PostCSS 5.x yet, so I had to manually remove that outdated package (i.e. `rm -r node_modules/postcss-import/node_modules/postcss`).¹ My project still seems to compile just fine afterwards.
- After doing so, `make test-watch` also succeeded, whereas before I just got some cryptic error.

¹ Thankfully, PostCSS emitted a helpful warning: "Your current PostCSS version is 5.0.1, but postcss-import uses 4.1.16. Perhaps this is the source of the error below."
